### PR TITLE
Style app - added styles to background and buttons

### PR DESCRIPTION
**Pull request by shradhaMe**
Please have a look and merge

### CODE REVIEW

**Comment by allisonvilla** [comment]
This looks really great! @shradhaMe my only suggestions would be
- do we want to use one of the brand colours for the default button colour (I love the yellow hover colour)
- also maybe move the transition from the :hover pseudo selector to the button selector itself so the transition works both ways 
- white background colour for the pokemon details box to better resemble the game (I've been using this vid as reference https://www.youtube.com/watch?v=Z22UP5btH_U - obviously shouldnt be exact) 

image

File: src/App.js [hunk 1]
```diff
@@ -3,13 +3,13 @@ import Footer from "./Footer";
...
-            <Header />
-            <DisplayGame />
-            <Footer />
-        </div>
-    );
+  return (
+    <div className="wrapper">
+      <Header />
+      <DisplayGame />
+      <Footer />
```
Comment (allisonvilla) [inline src/App.js:10]: I would move the footer from the wrapper so that it takes up the full width of the viewport

File: src/index.css [hunk 2]
```diff
@@ -262,6 +276,7 @@ h1 {
     padding: 20px 40px;
     width: 40%;
     margin: 30px 30px;
+    border-radius:20px;
 }
```
Comment (allisonvilla) [inline src/index.css:280]: We could also add a background-color of white to the .details-container to make it more distinct from the background?

**Review by allisonvilla — COMMENTED** [review]
lovely use of the brand colours @shradhaMe :) 

left some suggestions about making the details container look more distinct from the background colour and about the footer in this review also

File: src/App.js (hunk 1 above)
Comment (rachellehanna) [reply src/App.js:10]: :point_up_2:

File: src/index.css (hunk 2 above)
Comment (shradhaMe) [reply src/index.css:280]: Hi Allison, Will be add background-color of white to .details-container.

Comment (rachellehanna) [reply src/index.css:280]: I agree with Allison, it needs more contrast.

File: src/App.js (hunk 1 above)
Comment (shradhaMe) [reply src/App.js:10]: Hi Allison, if footer is removed from wrapper it does take up full width of the viewport but there is space below which if I used position:fixed hides our buttons. So do you still want it outside?

File: src/index.css [hunk 3]
```diff
@@ -276,12 +291,37 @@ h1 {
...
     text-transform: none;
 }
 
+.current-hand img{
+  padding:0 7px;
+}
+
+/* Button styles */
+
+button {
```
Comment (rachellehanna) [inline src/index.css:307]: I would add on the button
`margin-right: 1rem`

File: src/index.css [hunk 4]
```diff
@@ -276,12 +291,37 @@ h1 {
...
+transition:0.7s;
+cursor:pointer;
+}
+
+button:hover{
+  background-color: #ffff00;
+}
+
+button:focus{
+  background-color: #ffff00;
```
Comment (rachellehanna) [inline src/index.css:320]: In my opinion, the buttons should only be yellow when hovered over. I don't think they should remain yellow once they're clicked, it makes for confusing UX.

**Review by rachellehanna — COMMENTED** [review]
Looks good! 
I just made a few suggestions. I also think all the content, basically everything in `<section>` would look better centered, not off to the left.

File: src/App.js (hunk 1 above)
Comment (rachellehanna) [reply src/App.js:10]: I think it's an issue of margin collapsing. To begin with, I would get rid of the margin on the `p`.
I'm not sure `position: fixed` is necessary.

**Pull request merged**